packages/cf-builder-card/src/CardBuilder.js

```diff
@@ -48,21 +66,7 @@ class CardBuilder extends React.Component {
   }
 }
 
-CardBuilder.propTypes = {
```

Comment:
That's fine, we'd have to change the types anyway to take full advantage of Flow (i.e. functions). I'm just trying to understand if Flow requires us to use class properties.